Makefile

```diff
@@ -18,10 +18,10 @@ test:
 	go test
 
 test-all:
-	go test -tags=integration -v
+	go test -tags=integration
```

Comment:
`-v` flag is useful especially in the CI to see which tests ran.

Reply:
in the main pilosa Makefile, we do a TESTFLAGS variable to enable this if necessary. having it on by default is way too much output